skyportal/handlers/api/analysis.py

```diff
@@ -424,3 +467,510 @@ def delete(self, analysis_service_id):
         self.verify_and_commit()
 
         return self.success()
+
+
+class AnalysisHandler(BaseHandler):
+    def call_external_analysis_service(
```

Comment:
I might suggest that anything where you might think internally we might commit to the database you make just a normal function outside of the handler (as we had to for photometry). I could imagine some internal services using the framework to commit something into the AnalysisTable and it would be simpler if that commit was in a function callable from another handler.

Reply:
Good idea. Done.